src/export.ts

```diff
@@ -56,7 +56,8 @@ export const backupFromDoc = async <T>(
     for (const doc of docs) {
       const subCollections = await doc.ref.listCollections()
 
-      data[collectionName][doc.id] = doc.data()
+      const d = doc.data()
+      data[collectionName][doc.id] = d ? d : {}
```

Comment:
Thanks, @asankha. For typescript, we use the nullish coalescing operator instead of assigning a new constant. 

```
      data[collectionName][doc.id] = doc.data() ?? {}
````      

What do you think?

Reply:
Of course, please go ahead with the suggestion. I am new to typescript.